include/RAJA/exec-cuda/reduce_cuda.hxx

```diff
@@ -1032,7 +1037,7 @@ class ReduceMinLoc<cuda_reduce<BLOCK_SIZE>, T> {
           RAJA_MINLOC(sd[threadId],
                       m_blockdata[m_blockoffset + blockId + 1]);
       __threadfence();
-      int oldBlockCount = atomicAdd(&retiredBlocks[m_myID], (int)1);
+      unsigned int oldBlockCount = ::atomicAdd((unsigned int*)&retiredBlocks[m_myID], (unsigned int)1);
```

Comment:
This doesn't do anything to replace our CAS variants, the :: qualifier on this atomicAdd is to bring in the CUDA default one for unsigned int -- which I should have done in the first place.   Also, this was tested with 7.5 too.